core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -415,6 +418,18 @@ class AuthorizerIntegrationTest extends KafkaServerTestHarness {
   }
 
   @Test
+  def testPattrenSubscriptionWithTopicAndGroupRead() {
```

Comment:
Adding a negative test will probably be a good idea as well, if it is not there already.